snark-keys/src/error.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 ChainSafe Systems
```

Comment:
Please hold off on this module until we are ready to start the verification milestone. This stuff is changing underneath.